Documentation/gettingstarted/flannel-integration.rst

```diff
@@ -0,0 +1,127 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _flannel-integration:
+
+**************************************
+Cilium integration with Flannel (beta)
```

Comment:
We have used 'tech-preview' elsewhere - what is the agreed upon terminology for determining the maturity of these sorts of features? We should be consistent cc: @tgraf